docs/essentials/exceptions/exceptions-catch-specific-exceptions.md

```diff
@@ -57,109 +57,4 @@ public class Ex13
 }
 ```
 
-C++
-```
-using namespace System;
-
-public ref class Employee
-{
-public:
-    Employee()
-    {
-        emlevel = 0;
-    }
-
-    //Create employee level property.
-    property int Emlevel
-    {
-        int get()
-        {
-            return emlevel;
-        }
-        void set(int value)
-        {
-            emlevel = value;
-        }
-    }
-
-private:
-    int emlevel;
-};
-
-public ref class Ex13
-{
-public:
-    static void PromoteEmployee(Object^ emp)
-    {
-        //Cast object to Employee.
-        Employee^ e = (Employee^) emp;
-        // Increment employee level.
-        e->Emlevel++;
-    }
-
-    static void Main()
-    {
-        try
-        {
-            Object^ o = gcnew Employee();
-            DateTime^ newyears = gcnew DateTime(2001, 1, 1);
-            //Promote the new employee.
-            PromoteEmployee(o);
-            //Promote DateTime; results in InvalidCastException as newyears is not an employee instance.
-            PromoteEmployee(newyears);
-        }
-        catch (InvalidCastException^ e)
-        {
-            Console::WriteLine("Error passing data to PromoteEmployee method. " + e->Message);
-        }
-    }
-};
-
-int main()
-{
-    Ex13::Main();
-}
-```
-
-Visual Basic
-```
-Imports System
-
-Public Class Employee
-    'Create employee level property.
-    Public Property Emlevel As Integer
-        Get
-            Return emlevelValue
-        End Get
-        Set
-            emlevelValue = Value
-        End Set
-    End Property
-
-    Private emlevelValue As Integer = 0
-End Class
-
-Public Class Ex13
-    Public Shared Sub PromoteEmployee(emp As Object)
-        'Cast object to Employee.
-        Dim e As Employee = CType(emp, Employee)
-        ' Increment employee level.
-        e.Emlevel = e.Emlevel + 1
-    End Sub
-
-    Public Shared Sub Main()
-        Try
-            Dim o As Object = New Employee()
-            Dim newyears As New DateTime(2001, 1, 1)
-            'Promote the new employee.
-            PromoteEmployee(o)
-            'Promote DateTime; results in InvalidCastException as newyears is not an employee instance.
-            PromoteEmployee(newyears)
-        Catch e As InvalidCastException
-            Console.WriteLine("Error passing data to PromoteEmployee method. " + e.Message)
-        End Try
-    End Sub
-End Class
-```
-
-The common language runtime catches exceptions that are not caught by a catch block. Depending on how the runtime is configured, either a debug dialog box appears, or the program stops executing and a dialog box with exception information appears. For information about debugging, see [Debugging and Profiling Applications](https://msdn.microsoft.com/library/7fe0dd2y).
+The common language runtime catches exceptions that are not caught by a catch block. Depending on how the runtime is configured, either a debug dialog box appears, or the program stops executing and a dialog box with exception information appears.
```

Comment:
The dialog box is only valid on  Windows and maybe on OS X. We should also include that there will be an error printed out to the STDERR.